app/src/pages/SpanHeader.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from "react";
+import { graphql, useFragment } from "react-relay";
+
+import { Flex, Text } from "@phoenix/components";
+import { LatencyText } from "@phoenix/components/trace/LatencyText";
+import { SpanKindToken } from "@phoenix/components/trace/SpanKindToken";
+import { SpanStatusCodeIcon } from "@phoenix/components/trace/SpanStatusCodeIcon";
+import { TokenCount } from "@phoenix/components/trace/TokenCount";
+import { fullTimeFormatter } from "@phoenix/utils/timeFormatUtils";
+
+import { SpanHeader_span$key } from "./__generated__/SpanHeader_span.graphql";
+
+type SpanHeaderProps = {
+  span: SpanHeader_span$key;
+};
+export function SpanHeader(props: SpanHeaderProps) {
+  const span = useFragment(
+    graphql`
+      fragment SpanHeader_span on Span {
+        name
+        spanKind
+        code: statusCode
+        latencyMs
+        startTime
+        tokenCountPrompt
+        tokenCountCompletion
+        tokenCountTotal
+      }
+    `,
+    props.span
+  );
+
+  const startTime = useMemo<Date>(() => {
+    return new Date(span.startTime);
+  }, [span]);
```

Comment:
Technically we only care about the start time changing

Reply:
oh yeah, good catch